Setup.md

```diff
@@ -17,6 +17,8 @@ This document contains instructions on how to install and deploy the arXivDigest
       * `~/arxivdigest/config.json`
       * `/etc/arxivdigest/config.json`
       * `%cwd%/config.json`
+  5. Run the `init_topic_list.py` script in the `/scripts/` folder if you want an intitial list of general topics users can select from.
```

Comment:
"...if you want an intitial list of general topics users can select from." => "... to populate the database with an initial list of (computer science) topics users can select from."

Reply:
wrote general topics instead of computer science since there are not only computer science related topics in the topic list